examples/NeuralNetwork/create_data.jl

```diff
@@ -0,0 +1,41 @@
+using Pkg: Pkg
```

Comment:
This file is for generating the FashionMNIST data in a form that can be copy-pasted or linked in to a Dex file

Reply:
see #40

Reply:
see #40 
Previously i had the genrated code for the constants pasted in to the main file
but it made refreshing the web view take like multiple seconds while it reparsed the file
and github refused to render it